lib/symbols.js

```diff
@@ -0,0 +1,19 @@
+'use strict'
+
+const keys = {
+  childrenKey: Symbol('fastify.children'),
```

Comment:
Why are we not using the same pattern as Node core?
Eg: instead of `childrenKey`, we should use `kChildren`.

Reply:
I agree with @delvedor here. If it's only hard work, I don't mind making the changes myself. But if we want to leave them as they are, this looks ready to land.